### ISSUE TITLE
Can <named-item> altText have xml:lang?

### ISSUE THREAD

**Issue by aphillips** [w3c/ttml2#270]
https://w3c.github.io/ttml2/spec/ttml2.html#metadata-value-named-item

The `<named-item>` construct has an attribute `altText` intended to provide descriptive text primarily for images (similar to `alt` in HTML). There is no associated language metadata. Should there be?

**Comment by nigelmegitt** [comment]
I don't think I understand the issue here - the `ttm:item` element permits `xml:lang` to be specified, so an example like:

```xml
<ttm:item xml:lang="en" name="altText">A heart</ttm:item>
<ttm:item xml:lang="fr" name="altText">Un coeur</ttm:item>
```

is specified and possible.  Is something extra needed?

**Comment by aphillips** [comment]
This may be a bit too much shorthand on our part. A common problem in markup languages is when an attribute can contain natural language text... for cases where the attribute and element contain different languages.

```
<ttm:item xml:lang="zxx" name="Book ISBN">976-0-12345678-X</ttm:item>
<ttm:item xml:lang="en" name="Un coeur">A heart</ttm:item>
```

**Comment by nigelmegitt** [comment]
Is that a usage problem or a specification problem? I'm still unclear exactly what the issue is here: what kind of resolution are you looking for?

**Comment by palemieux** [comment]
> A common problem in markup languages is when an attribute can contain natural language text...

@aphillips The `name` attribute of the `ttm:item` element is not free-form: it is a symbol from the constrained vocabulary of Section 14.3.2. Specifically, instead of defining _X_ XML elements, each with a tag equal to a value of the constrained vocabulary of Section 14.3.2, the specification defines a single `ttm:item` element with a `name` attribute that contains a symbol from the constrained vocabulary of Section 14.3.2. There is therefore no need to have a language associated with the `name` attribute. Makes sense, or did you have something else in mind?

**Comment by skynavga** [comment]
I concur that no action is required.

**Comment by palemieux** [comment]
Do not close issues that are not assigned to you.

**Comment by nigelmegitt** [comment]
This issue has been correctly labelled as a horizontal review comment; since horizontal review is essentially part of wide review and this comment came from outside the WG we should not be closing this issue without:

1. Agreeing a WG disposition
2. Confirming this disposition with the person who raised this issue.

Closing it without either of the above two things happening is inappropriate - @skynavga in future please take care when closing issues that it is appropriate to do so.

**Comment by nigelmegitt** [comment]
@aphillips If you could have a look at https://github.com/w3c/ttml2/issues/270#issuecomment-297746585 and respond that would be very helpful: if you concur with the other presented views that no change is needed here then as the original raiser of the issue it would be reasonable for you to add a comment to that effect and if you want, close the issue. Otherwise it is likely to be the group's disposition that no change is needed in this case, and we'll come back and ask you if that is acceptable in any case!

**Comment by skynavga** [comment]
Absent a technical reason not to close this (or any) issue, it is entirely appropriate to do so.

**Comment by aphillips** [comment]
I apologize for having typed the example incorrectly. The attribute in question is not `name`, it is `altText` (as the title says). This attribute is described in 14.3.2 as:

> altText
>    A string that expresses alternate text content, where the value adheres to xsd:string. Typically used to provide a textual equivalent or summary for some related image content.

This is natural language text. Any natural language text should have a way to indicate the language and direction for presentation and processing purposes.

**Comment by skynavga** [comment]
@aphhillips and there is a way to indicate language, namely, by using xml:lang, e.g.,

<ttm:item name="altText" xml:lang="fr">Il n'y a pas de problème ici</ttm:item>

there is no attribute here that contains natural language text

**Comment by nigelmegitt** [comment]
Thank you for the speedy response @aphillips - does https://github.com/w3c/ttml2/issues/270#issuecomment-282738598 not address that point about `altText` however?

**Comment by nigelmegitt** [comment]
@skynavga This isn't really the place to be discussing this, but the Process absolutely does require confirmation of disposition, see https://www.w3.org/2017/Process-20170301/#formal-address .

**Comment by aphillips** [comment]
@nigelmegitt If that's the syntax, then yes it does. Thanks.

**Comment by nigelmegitt** [comment]
Thank you @aphillips that is indeed the syntax; I'm going to take that as agreement from you that what you're seeking can already be done and that there is indeed no issue here, and therefore close the issue on your behalf - if I have misunderstood then please reopen it.

**Comment by aphillips** [comment]
@nigelmegitt "I am satisified" :-)

**Comment by skynavga** [comment]
@nigelmegitt regarding
>This isn't really the place to be discussing this, but the Process absolutely does require confirmation of disposition, see https://www.w3.org/2017/Process-20170301/#formal-address

Firstly, the essence of the process requirement
>a group has formally addressed an issue when it has sent a public, substantive response to the reviewer who raised the issue

is effectively satisfied by the above comments pointing out that there is no technical issue here. Secondly, there is no process requirement that requires the commenter to agree with the response. Thirdly, there is no process requirement that requirement that dictates who may close or when an issue is closed in our github repositories, whether the issue originated in a wide review, a horizontal review, or otherwise.

In other words, AFAIK, the github repo issue database is not equivalent to a disposition of comments document.

But let's not continue this thread in the present context, and defer it instead to an upcoming call or meeting.

**Comment by nigelmegitt** [comment]
For reference, an example corresponding to https://github.com/w3c/ttml2/issues/270#issuecomment-282738598 is added by #307 .

**Comment by r12a** [comment]
Thank you for considering this issue. The i18n WG is now closing the pointer to this issue in its review tracker.
